src/HasBinaryUuid.php

```diff
@@ -73,11 +76,19 @@ public static function decodeUuid(string $binaryUuid): string
 
     public function toArray()
     {
+        if (! $this->exists) {
+            return parent::toArray();
+        }
+
         return array_merge(parent::toArray(), [$this->getKeyName() => $this->uuid_text]);
     }
 
-    public function getUuidTextAttribute(): string
```

Comment:
This method should be type hinted as a nullable string:

```php
public function getUuidTextAttribute(): ?string
```

Reply:
Nullable types are not available in PHP 7.0. Do you want me to change requirement in the `composer.json` file?

Reply:
Sorry I missed that, yes feel free to bump the PHP version!

Reply:
Done :+1: